rector.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
+use Rector\Config\RectorConfig;
+use Rector\Doctrine\Set\DoctrineSetList;
+use Rector\Set\ValueObject\LevelSetList;
+use Rector\PHPUnit\Set\PHPUnitSetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+        __DIR__ . '/tests-unit',
+    ]);
+
+    // register a single rule
+    $rectorConfig->rule(InlineConstructorDefaultToPropertyRector::class);
+
+    $rectorConfig->sets([
+        PHPUnitSetList::PHPUNIT_91,
+        DoctrineSetList::DOCTRINE_DBAL_30,
+        DoctrineSetList::DOCTRINE_DBAL_211,
```

Comment:
Mohls ještě pustit třeba CODE_QUALITY, DEAD_CODE, NAMING, TYPE_DECLARATION ať je sranda :D

Reply:
No, code quality našlo ještě jeden case deprekované metody query vs executeQuery...

Reply:
ca77b14d44d27d83b4a4387c8b00f1bbf9121695

Reply:
Budem tomu věřit nebo mám pustit tag? :D

Reply:
nic moc, čekal sem víc, asi je to dost kvalitní, to co tam je je imho z toho DOCTRINE_DBAL_30